### PR TITLE
updated readme

### PR DESCRIPTION
**Pull request by stacyburris**
- [x] Top-level README “Table of Contents” is updated
 - [x] Feature tasks for this challenge are completed
 - [x] Unit tests written and passing
     - [x] “Happy Path” - Expected outcome
     - [x] Expected failure
     - [x] Edge Case (if applicable/obvious)
 - [x] README for this challenge is complete
     - [x] Summary, Description, Approach & Efficiency, Solution
     - [x] Link to code
     - [ ] Picture of whiteboard

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,3 +22,4 @@
 + [leftJoin](javascript/code-challenges/leftJoin/README.md)
 + [graph](javascript/code-challenges/graph/README.md)
 + [breadthFirst](javascript/code-challenges/breadthFirst/README.md)
++ [getEdge](javascript/code-challenges/getEdge/README.md)
```
Comment (adrianhuebner) [inline README.md:25]: Everything looks good just need to see tests added!
